commit 4a92bb1519535cc3aad9b996a5392a79568e9943 (refs/changes/60/160/1)
Author: Joshua M. Clulow <jmc@joyent.com>
Date:   2016-07-27T17:09:37-07:00 (3 years, 2 months ago)
    
    HEAD-2317 sdc-headnode "make check" should work without SSH keys

diff --git a/package.json b/package.json
index ce8aee52..0eaf82cd 100644
--- a/package.json
+++ b/package.json
@@ -10,7 +10,7 @@
     "extsprintf": "^1.3.0",
     "jscs": "^1.13.1",
     "jshint": "^2.7.0",
-    "manta": "git+ssh://git@github.com:joyent/node-manta.git#12f4c84",
+    "manta": "git+https://github.com/joyent/node-manta.git#12f4c84",
     "monowrap": "^1.0.4",
     "progbar": "^1.1.0",
     "readtoend": "0.0.1",
